Task3HW3qsortAndSearchOfMostFrequent.c

```diff
@@ -0,0 +1,111 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
```

Comment:
```suggestion
#include <time.h>

```
После include нужен перенос строки (как и после всех define, после каждой функции.